src/sailfishui/crash-reporter.qml

```diff
@@ -111,35 +110,33 @@ Page {
                 onClicked: PrivacySettings.autoDeleteDuplicates = !PrivacySettings.autoDeleteDuplicates
             }
 
-            SystemdServiceSwitch {
-                serviceName: "crash-reporter-endurance.service"
-                managerType: SystemdService.SystemManager
-
+            TextSwitch {
                 //% "Endurance reports"
                 text: qsTrId("settings_crash-reporter_enable_endurance")
                 //% "Reports system statistics helping diagnose problems that "
                 //% "manifest themselves only after a long-term use of the "
                 //% "device like memory leaks, excessive battery drain, or "
                 //% "decreasing performance."
                 description: qsTrId("settings_crash-reporter_enable_endurance_description")
-
-                onBeforeStateChange: {
-                    PrivacySettings.endurance = newState
+                automaticCheck: false
```

Comment:
Initial state is broken now (same for Journal Spy)

Add...
```
checked: PrivacySettings.endurance
```

Reply:
doh :)